server/src/main/java/org/candlepin/resource/HypervisorResource.java

```diff
@@ -186,6 +190,53 @@ else if (guestIdsUpdated) {
         return result;
     }
 
+    /**
+     * Creates or Updates the list of Hypervisor hosts
+     * <p>
+     * Allows agents such as virt-who to update hosts' information . This is typically
+     * used when a host is unable to register to candlepin via subscription manager.
+     * <p>
+     * In situations where consumers already exist it is probably best not to
+     * allow creation of new hypervisor consumers.  Most consumers do not
+     * have a hypervisorId attribute, so that should be added manually
+     * when necessary by the management environment.
+     *
+     * @param hypervisorJson the json representation of the hypervisors
+     * @param principal
+     * @param ownerKey key of owner to update
+     * @param createMissing specify whether or not to create missing hypervisors.
+     * Default is true.  If false is specified, hypervisorIds that are not found
+     * will result in a failed state of the job.
+     * @return a JobDetail object
+     *
+     * @httpcode 202
+     * @httpcode 200
+     *
+     */
+    @POST
+    @Consumes(MediaType.TEXT_PLAIN)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Transactional
+    @Path("/{owner}")
+    public JobDetail updateOwnersHypervisors(
```

Comment:
Lets mark the old method deprecated, and could you name them somewhat similar so it's a bit more clear that they're the same functionality just a newer version. Something like hypervisorUpdate and hypervisorUpdateAsync perhaps.